_docs/firecloud-launch-with.md

```diff
@@ -12,6 +12,8 @@ information on what that looks like from a user point of view in a mini tutorial
 When browsing WDL workflows from within Dockstore, you will see a "Launch with FireCloud" button on the right. The currently selected
 version of the workflow will be exported, but this can subsequently be changed in the FireCloud UI if need be.
 
+Note: Workflows with file-based imports are unsupported in FireCloud, the button will be disabled for those workflow versions.
```

Comment:
Same here, please elaborate and show an example of how one might transform a file based import into a http import as a workaround. (you might be able to use some content from topmed).